index.js

```diff
@@ -1,41 +1,28 @@
 const express = require('express');
 const knex = require('knex');
+const db_config = require('./knexfile.js');
+const db = knex(db_config.development);
 
-const dbConfig = require('./knexfile');
+const projectRouter = require('./routers/projectRouters');
+const actionRouter = require('./routers/actionRouters');
 
 const server = express();
-const db = (dbConfig.development);
-const PORT = 0112
+const PORT = 5000;
 
 server.use(express.json());
-
-server.post('/project', (req, res) => {
-    const project = req.body;
-    db('project').insert(project)
-        .then(project => {
-            res.status(201).json(project);
-        })
-        .catch(err => {
-            res.status(500).json({ err: 'Failed to insert project' })
-        });
-});
-
-server.post('/action', (req, res) => {
-    const action = req.body;
-    db('action').insert(action)
-        .then(action => {
-            res.status(201).json(action);
-        })
-        .catch(err => {
-            res.status(500).json({ err: 'Failed to insert action' })
-        });
-});
+server.use('/project', projectRouter);
+server.use('./action', actionRouter);
 
 server.get('/project/:id', (req, res) => {
-    const { id } = req.params;
-    db('project').where('id', id)
+    const { id } = req.params
+    db('project')
+        .first()
```

Comment:
`db('project')` is getting all of the projects, and then `first()` is just getting the first one in that array. This endpoint will work as expected for the first project, but not for any others (as it'll be displaying the first project with the desired project's actions, and there's certainly a disconnect there).